src/framework/components/animation/system.js

```diff
@@ -56,7 +56,7 @@ Object.assign(pc, function () {
             var key;
             this.addComponent(clone, {});
 
-            clone.animation.data.assets = pc.extend([], entity.animation.assets);
+            clone.animation.assets = entity.animation.assets.slice();
```

Comment:
Only thing here is is there a chance `assets` can be null?

Reply:
I don't think so but @daredevildave can confirm.